src/components/DisplayNames/index.js

```diff
@@ -23,15 +20,14 @@ class OptionRowTitle extends PureComponent {
 
     componentDidMount() {
         this.setState({
-            isEllipsisActive: this.containerRef && hasEllipsis(this.containerRef),
+            isEllipsisActive: this.containerRef && hasContentWiderThanScrollWidth(this.containerRef),
```

Comment:
@marcaaron This function is only used once, right here? Should we just inline the logic here instead of having a library function with questionable reuse potential?

Reply:
I would suggest combining multiple small utilities into one file.

Reply:
Yeah I think making a separate utility here is flatly against our engineering best practice recommendations to not over engineer things.

That said, if we have a bunch of utilities that do nothing but measure react refs we could group them all somewhere. But not sure if/when that will happen.